docs/kratos/mfa/15_totp.mdx

```diff
@@ -112,23 +88,100 @@ In this example, the user identifier (email - `alice@example.com`) is used. To l
    ory update identity-config <project-id> --file identity-config.yaml
    ```
 
+4. After TOTP has been enabled for the Ory Network project, a user can enable TOTP for their own account by signing in and going
+   to the **Authenticator App** section of the **Account Settings** page (see
+   [Pairing a TOTP app with your account](#pairing-a-totp-app-with-your-account)).
+
+   :::note
+
+   In the Ory Account Experience, users are **not** automatically prompted to enable TOTP on their account.
+
+   :::
+
 ```mdx-code-block
 </TabItem>
 </Tabs>
 ```
 
-### Distinguishing identities requesting TOTP
+## Account Experience with TOTP
 
-To help users distinguish which identity the registered TOTP code belongs to, add a `totp` section to the identity schema section
-of the trait you want to show up in the TOTP app.
+After TOTP has been enabled in the Ory Network project, users have the ability to manage their own TOTP configuration using
+**Account Settings**.
+
+### Pairing a TOTP app with your account
+
+If TOTP is enabled in Ory Network, you can pair a TOTP app with your account, as follows:
```

Comment:
```suggestion
If TOTP is enabled in your Ory Network project, you can pair a TOTP app with your account, as follows:
```

Reply:
I guess the `you` afterward doesn't fit well now. Maybe it should also be `a user can pair...`?

Reply:
Changed "your Ory Network project" to "the Ory Network project". That way, the remaining "yous" in this section are consistent with each other (addressing the user).